index.html

```diff
@@ -40,18 +40,18 @@
              </div>
          </nav>
 
-         <nav class="mobile-navigation">
-          <ul>
+         <nav class="mobile-navigation" >
+          <ul onclick="display()">
             <li>
-                <a href="#">Portfolio</a>
+                <a href="#portfolio">Portfolio</a>
             </li>
 
             <li>
-                <a href="#">About</a>
+                <a href="#about">About</a>
             </li>
 
             <li>
-                <a href="#">Contact</a>
+                <a href="#contact">Contact</a>
             </li>
```

Comment:
- [ ] [OPTIONAL] Please make sure the navlinks are visible in the desktop version, currently it is not visible, I think it is because currently the nav links color is set to be white please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210421217-d83cf1f6-2af7-4450-9805-e26686006332.png)